main/src/com/google/refine/expr/functions/ToDate.java

```diff
@@ -187,15 +196,26 @@ private OffsetDateTime parse(String o1, DateFormat formatter) {
             return null;
         }
     }
+
+    /**
+     * Function for parsing OffsetDateTime from unix time format (seconds since 1 January 1970 00:00:00 UTC)
+     */
+    private OffsetDateTime parse(String o1)
+    {
+        Long seconds = Long.parseLong(o1);
+        Instant instant = Instant.ofEpochSecond(seconds);
+        OffsetDateTime date = instant.atOffset(ZoneOffset.UTC);
+        return date;
+    }
     
     @Override
     public String getDescription() {
-        return "Returns o converted to a date object, you can hint if the day or the month is listed first, or give an ordered list of possible formats using this syntax: http://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html";
+        return "Returns o converted to a date object, you can hint if the day or the month is listed first, use unix time (with \"unix\" flag) or give an ordered list of possible formats using this syntax: http://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html";
     }
     
     @Override
     public String getParams() {
-        return "o, boolean month_first / format1, format2, ... (all optional)";
+        return "o / integer unix_seconds (only with unix flag), boolean month_first / \"unix\" / format1, format2, ... (all optional) ";
```

Comment:
I don't think " / integer unix_seconds (only with unix flag)" is should be here - it is just another object. This is about the behaviour of the function and should be in the Description not the parameters

Reply:
Also based on my reading of the code I think integers will be treated as Year if unix flag not set?

Reply:
Yes. Based on comment in line 78 I think it's a desired behaviour.